engine/execution/messages.go

```diff
@@ -52,3 +52,61 @@ func NewEmptyComputationResult(
 		},
 	}
 }
+
+func (cr ComputationResult) TxResByColIndex(colIndex int) []flow.TransactionResult {
```

Comment:
This would be more readable with a full name, or at least a comment.

Reply:
updated the name, also made it private